Add a version command

katekligman opened the pull request:
Adds a version command to Wraith.  Fix for #397.

When the command ‘wraith version’ is run it will output ‘Wraith version X.X.X’

ChrisBAshton commented on lib/wraith/cli.rb:
```diff
@@ -176,4 +176,9 @@ def latest(config)
+
+  desc "version", "Show the version of wraith"
+  def version
+    puts "Wraith version " + Wraith::VERSION
```
I would change this to `logger.info Wraith::VERSION` just to keep this consistent with the rest of the codebase (and not sure the "Wraith version" bit is necessary, other programs just output their version numbers).

ChrisBAshton commented:
Thanks for the PR, I wonder if this could work with `wraith --version` like other programs?

i.e.
```
$ phantomjs version
Can't open 'version'

$ phantomjs -version
Error: Unknown option: e

$ phantomjs --version
1.9.7
```

I agree Wraith should have a version command, just not sure `wraith version` is the 'standard' way of invoking it :)

katekligman commented:
Thanks for the feedback!

I've changed to logger.info and removed the 'Wraith version' from the output.

For the version command itself: I've noticed other programs tend to stay somewhat consistent with the formats of their parameters. phantomjs uses '--' with other parameters so it would make sense for their version command to also have that prefix. 

That said, there are a lot of variations among other programs, so I kept 'version' as the primary Wraith documented command, to stay consistent with Wraith's other commands which don't use prefixes, and I also put in a mapping so "-v", "--version" and "-version" will also invoke it.

ChrisBAshton commented:
Fantastic!

It would also be really helpful if you could make a PR to the gh-pages branch so that the new command is described in http://bbc-news.github.io/wraith/#Commandlineoptions. If not, I'll get around to it eventually!

Thanks once again.
